.circleci/config.yml

```diff
@@ -1,29 +1,56 @@
 version: 2
 jobs:
+  integration-php72-magento23:
+    docker:
+      - image: boltdev/m2-plugin-ci-integration-php72:v0
+        auth: 
```

Comment:
could you clarify what you mean by install here?

Reply:
talked with Ethan offline, I was wondering why below we didn't need auth and here we need. Now I understand this one is private docker image because it contains magento related credentials